src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/util/GraphQLRequestParamUtils.java

```diff
@@ -46,16 +48,18 @@
  */
 public final class GraphQLRequestParamUtils {
 
-    private static final String VARIABLES_FIELD = "variables";
+    static final String VARIABLES_FIELD = "variables";
```

Comment:
Should this be public then?

Reply:
Makes sense. Done: ff3eba435347a3148332aa22a0a6d6d0e08646ff